build.sbt

```diff
@@ -0,0 +1,206 @@
+import com.amazonaws.auth.{EnvironmentVariableCredentialsProvider, InstanceProfileCredentialsProvider}
```

Comment:
Just curious: Are there any advantages of using build.sbt instead of scala based?